source-code-flow-manager-gitlab/src/main/java/com/iprody/source/code/flow/manager/api/job/PipelineJobAPI.java

```diff
@@ -0,0 +1,20 @@
+package com.iprody.source.code.flow.manager.api.job;
+
+import com.iprody.source.code.flow.manager.api.job.domain.Job;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Flux;
+
+public class PipelineJobAPI {
+    private WebClient client;
+    public PipelineJobAPI(WebClient client) {
```

Comment:
Empty line between statements 
```suggestion

    public PipelineJobAPI(WebClient client) {
```